apps/aehttp/test/aehttp_parent_connector_tests.erl

```diff
@@ -10,7 +10,8 @@
 -define(CHILD_CHAIN_NETWORK_ID, <<"hc_network_id">>).
 -define(SIGN_MODULE, aec_preset_keys).
 
-ae_sim_test_() ->
+% removed as test, should/could be removed altogether, same goes for btc version below.
+ae_sim_test_rem() ->
```

Comment:
`test_` as ending of a function name has a specific meaning. Cannot just at `rem` after it without changing that meaning. See eunit documentation for detail

Reply:
I know, this leaves that particular test out of the suite. So here I could keep that code, in case it was actually testing something besides commitments. But perhaps it is not?

Reply:
So, my question is if the two "_rem" methods (that are left in the code but not invoked as tests...) actually do anything but commitment testing, or of they can be removed altogether?

Reply:
You want to write very similar tests as soon as you add pinning, right?